### PR TITLE
C33629: Verify My Number button is inactive with invalid phone number

### PR DESCRIPTION
**Pull request by angy113**

### CODE REVIEW

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Settings/SupportLayout.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.jalasoft.callingapp.ui.layouts.pages.Settings;
+
+import com.jalasoft.callingapp.ui.layouts.pages.Login.LoginLayout;
+import com.jalasoft.ui.layouts.BaseLayout;
+import com.jalasoft.utils.CommonActions;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class SupportLayout extends BaseLayout {
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Settings/SupportLayout.java]: Add doc for the class

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java [hunk 2]
```diff
@@ -46,4 +51,25 @@ public void iLogInToTheApplication(final Map<String, String> loginCredentialsMap
...
     }
+
+    /**
+     * Logout from the application if user is logged.
+     */
+    @Given("^I am not login to the application$")
+    public void iAmNotLoginToTheApplication() {
+        if (!loginLayout.isDisplayed()) {
+            BaseAppLayout baseAppLayout= new BaseAppLayout();
+            loginLayout = baseAppLayout.clickOnSettingsIcon()
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java]: Create a method to logout in BaseAppLayout class

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java [hunk 3]
```diff
@@ -46,4 +51,25 @@ public void iLogInToTheApplication(final Map<String, String> loginCredentialsMap
...
+                    .clickOnLogoutBtn();
+        }
+    }
+
+    /**
+     * Verifies if " Verify Mu Number " is disabled.
+     */
+    @Then("^The Verify My Number button should be disabled$")
+    public void theVerifyMyNumberButtonShouldBeDisabled() {
+       assertFalse(loginLayout.isVerifyMyNumberButtonEnabled());
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java]: change to the format:
assertFalse(validation, "message");

File: IDT.CallingApp/src/test/resources/Login.feature [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  As regular user with bolivia account
+  I want to know if login section is correctly working
+
+  Scenario: C33629: Verify My Number button is inactive with invalid phone number
+    Given I am not login to the application
+    When I login to the application with the following credentials
+        | Phone   | 0                                    |
+        | Country | Bolivia, Plurinational State Of (BO) |
+        | Search  | Bolivia                              |
+    Then The Verify My Number button should be disabled
```
Comment (wMercadoJala) [inline IDT.CallingApp/src/test/resources/Login.feature]: add empty line to the en of file

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java (hunk 2 above)
Comment (angy113) [reply IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java]: Done

File: IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Settings/SupportLayout.java (hunk 1 above)
Comment (angy113) [reply IDT.CallingApp/src/main/java/com/jalasoft/callingapp/ui/layouts/pages/Settings/SupportLayout.java]: Done

File: IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java (hunk 3 above)
Comment (angy113) [reply IDT.CallingApp/src/test/java/com/jalasoft/callingapp/steps/LoginSteps.java]: Done

File: IDT.CallingApp/src/test/resources/Login.feature (hunk 4 above)
Comment (angy113) [reply IDT.CallingApp/src/test/resources/Login.feature]: Done

**Pull request merged**